monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt_br.md

```diff
@@ -0,0 +1,78 @@
+
+Digite [@sunchakr](https://twitter.com/sunchakr), uma engenheira nova-iorquina revolucionária envolvida no projeto Monero que se interessa e se compromete com blockchain desde 2015. Aqui ela fala com Monica Mizzi sobre sua experiência nessa indústria – tudo, desde seu primeiro contato com o mundo do blockchain, passando pelas suas considerações sobre fazer parte da comunidade Monero, até o seu comprometimento com o tópico “mulheres no blockchain”.
+
+**Oi @sunchakr! Muito obrigada por concordar em responder algumas perguntas sobre sua experiência no espaço blockchain e Cripto. Vamos começar! Como você se envolveu nessa indústria? O que chamou sua atenção?**
+
+Um amigo meu que estava lendo sobre o assunto e acompanhando a tecnologia de perto me apresentou o Bitcoin em 2015. Achei o tema interessante, mas estava confusa sobre como funcionava. Depois que entendi a tecnologia, fui capaz de assimilar o conceito por trás dessa ideia revolucionária e percebi seu enorme potencial.
+
+Esse produto pode mudar as nossas vidas e o mundo de tantas maneiras - é possível evitar essas violações de dados “acidentais” que vazam todas as suas informações pessoais, tornando você um alvo de marketing, anúncios e hackers; o controle governamental de suas finanças, como o que está acontecendo na Venezuela, ou o controle dos bancos sobre transações cotidianas, os quais têm acesso a uma simples transferência de fundos de uma conta para outra no fim de semana.
+
+A maioria de nós não percebe que nós não temos controle do nosso dia-a-dia e que muitos itens essenciais são controlados por outras pessoas e não há como evitar isso. Nosso sistema foi projetado dessa forma para que os “poderosos” possam controlar os “vulneráveis”. Acredito que esta "Revolução da Criptomoeda" surgiu da necessidade de tentar recuperar o poder, resistindo à censura. As pessoas deveriam ter autoridade para administrar suas finanças, bem como distribuir a riqueza dentro das sociedades por meio desse sistema ponto a ponto de dinheiro digital, sem que um intermediário assuma o controle.
+
+**Você esteve envolvida no projeto Monero por um bom tempo. Você pode compartilhar alguns detalhes sobre o seu envolvimento, sobre quando e como começou e o que faz?**
+
+Eu ingressei no Monero no final de 2016. Eu conheci muitas moedas alternativas e percebi que Monero não era como as outras. Comecei a trabalhar em projetos relacionados ao Monero no início deste ano, criando o primeiro Monero Meetup em Nova York. Na época só víamos encontros de blockchain ou bitcoin, então nosso grupo - incluindo [@chowbungaman](https://twitter.com/chowbungaman) e [Vik Sharma](https://www.youtube.com/watch?v=joAZQXp69Kw), fundador do [Cake Wallet](https://cakewallet.io/), a primeira Carteira Monero para iOS - decidiu iniciar um Monero Enthusiast Meetup. Tivemos um público maravilhoso e o grupo continua a prosperar.

```

Comment:
Use female gender here: "fundador d**a** [Cake Wallet]" because we are referring to "carteira".